src/lib/gql/mutations.ts

```diff
@@ -311,6 +312,32 @@ export async function updateTeammates(circleId: number, teammates: number[]) {
   return updateTeammates;
 }
 
+export async function updateAllocations(
+  circleId: number,
+  params: PostTokenGiftsParam[]
+) {
+  await client.mutate({
+    updateAllocations: [
+      {
+        payload: {
+          circle_id: circleId,
+          allocations: params.map(a => {
+            const withPatchedNote: Omit<typeof a, 'note'> & { note: string } = {
+              ...a,
+              note: a.note ? a.note : '',
```

Comment:
If you do this, do you still need the Omit?
```suggestion
              note: a.note ? a.note.toString() : '',
```

Reply:
@META-DREAMER I think its even simpler, like this

```
return {
  ...a,
  note: a.note ? a.note : '',
};
```